commerce_coordinator/apps/titan/clients.py

```diff
@@ -223,6 +223,59 @@ def update_payment(self, payment_number, payment_state, response_code):
 
         return response['data']['attributes']
 
+    def create_payment(
+        self,
+        order_uuid,
+        response_code,
+        payment_method_name,
+        provider_response_body,
+        reference=None,
+        amount=None,
+        payment_date=None,
+        source=None,
+    ):
+        """
+        Request Titan to Create payment.
+
+        Args:
+            order_uuid(str): Order UUID related to this order.
+            response_code(str): Payment attempt response code (payment intent id) provided by stripe.
+            payment_method_name(str): The name of the payment method used for this payment. See enums for valid values.
+            provider_response_body(str): The response JSON dump from a request to the payment provider..
+            reference(str): Optional. Reference to be saved against the payment.
+            amount(int): Optional. Payment amount to apply to order.
```

Comment:
[upstream spec question] I just noticed `amount` is an integer. Does this mean this is a number of cents?

Reply:
Ah nice catch thanks, will fix that, should be a float

Reply:
Good. I am updating this type to float. (We need to update the spec accordling @RylanMorrison)